documentation/book/assembly-custom-resources.adoc

```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies:
+//
+// getting-started.adoc
+
+[id='assembly-custom-resources-{context}']
+= Custom resources
+Custom Resource Definitions (CRDs) extend the Kubernetes API, providing definitions to create and add custom resources to an {ProductPlatformName} cluster. Custom resources are created as instances of CRDs.
+
+For {ProductName}, CRDs introduce custom resources specific to {ProductName} to an {ProductPlatformName} cluster, such as Kafka, Kafka Connect, Kafka Mirror Maker, and users and topics.  CRDs provide configuration instructions, defining the schemas used to instantiate and manage the resources specific to {ProductName}. CRDs also allow {ProductName} resources to benefit from native {ProductPlatformName} features like CLI accessibility and configuration validation.
+
+CRDs require a one-time installation to a cluster. Depending on the cluster setup, installation typically requires cluster admin privileges.
+
+NOTE: Access to manage custom resources is limited to xref:assembly-getting-started-strimzi-admin-{context}[{ProductName} administrators].
+
+CRDs and custom resources are defined as YAML files.
+
+A CRD defines a new `kind` of resource — such as `kind:kafka` — within a {ProductPlatformName} cluster.
```

Comment:
Travis is complaining about this ... I think that `{ProductPlatformName}` always starts with OpenShift.

```suggestion
A CRD defines a new `kind` of resource — such as `kind:kafka` — within an {ProductPlatformName} cluster.
```